lumigator/python/mzai/backend/backend/services/datasets.py

```diff
@@ -83,6 +84,15 @@ def validate_experiment_dataset(filename: str):
             )
 
 
+def dataset_has_gt(filename: str) -> bool:
```

Comment:
we should pass in a complete path here and then pass that into the with open

```
with open(filename, r) as f
```

are we also assuming that the file is a csv here? we might want to specify or do a type check before we open/assign

Reply:
I am using the same identical interface of `validate_experiment_dataset` here for consistency.
We assume the file is a CSV as we have already checked it in `validate_dataset_format`.